### PR TITLE
roof-removal: both of juliet's balconies are now hidden at level 0.

### PR DESCRIPTION
**Pull request by arnaudperalta**
Closes https://github.com/runelite/runelite/issues/14098

Two balconies weren't hidden at Juliet's house.

### CODE REVIEW

File: runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc [hunk 1]
```diff
@@ -513,6 +513,24 @@
       "z2": 0
     }
   ],
+  "12597": [ //Juliet's House
```
Comment (Hydrox6) [inline runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:516]: This region is not just Juliet's House. Would be more sensible to call it `Varrock West`

Comment (Hydrox6) [reply runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:516]: Also why is this in the middle of the list?

File: runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc [hunk 2]
```diff
@@ -513,6 +513,24 @@
...
+      "z2": 0
+    },
+    {
+      "rx1": 29,
+      "ry1": 37,
+      "rx2": 30,
+      "ry2": 45,
+      "z1": 0,
+      "z2": 0
+    }
```
Comment (Hydrox6) [inline runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:532]:
```suggestion
    {
      "rx1": 29,
      "ry1": 38,
      "rx2": 30,
      "ry2": 44,
      "z1": 0,
      "z2": 0
    }
```

**Review by Hydrox6 — CHANGES_REQUESTED** [review]

File: runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc (hunk 1 above)
Comment (arnaudperalta) [reply runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:516]: Because it is after "12596" in the list

Comment (Hydrox6) [reply runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:516]: There is no defined order to the file, so putting it in the middle is just really weird

Comment (arnaudperalta) [reply runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:516]: I'll change that, thanks for the review

**Pull request merged**